source/ToSort/XSharp.Sdk/build/XSharp.Sdk.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
 
-    <Import Project="$(MSBuildThisFileDirectory)..\tools\xsc\build\XSharp.Compiler.props" />
+    <Import Project="$(MSBuildThisFileDirectory)..\..\XSharp\XSC\build\XSharp.Compiler.props" />
```

Comment:
This should point to the compiler included in the SDK (the path that was there). To test locally, you can for example add a property `XSharpCompilerPropsPath`:
```xml
<PropertyGroup>
    <XSharpCompilerPropsPath Condition="'$(XSharpCompilerPropsPath)' == ''"></XSharpCompilerPropsPath>
</PropertyGroup>
```

and then:
```xml
<Import Project="$(XSharpCompilerPropsPath)" Condition="'$(XSharpCompilerPropsPath)' != '' AND Exists('$(XSharpCompilerPropsPath)')" />
```

In the test projects (for example `XSharp.Playground.csproj`) you just need to set the path:

```xml
<XSharpCompilerPropsPath>$(MSBuildThisFileDirectory)..\..\XSharp\XSC\build\XSharp.Compiler.props</XSharpCompilerPropsPath>
```